Makefile

```diff
@@ -405,24 +405,25 @@ unit-test-coverage:
 tidy:
 	$(eval MIN_GO_VERSION := $(shell grep -Eo '^go\s+[0-9]+\.[0-9.]+' go.mod | cut -d' ' -f2))
 	$(GO) mod tidy -compat=$(MIN_GO_VERSION)
+	@$(MAKE) --no-print-directory $(GO_LICENSE_FILE)
 
-.PHONY: vendor
-vendor: tidy
+vendor: go.mod go.sum
```

Comment:
is `make vendor` still running if vendor folder exists?

Reply:
It would only run if the timestamp on the folder is older than go.mod and go.sum.

Reply:
Well i can ajust my scripts ...

Reply:
I guess I can revert it and add a comment why it's `.PHONY`. To me it seemed like a bug because a target that produces a file should not be phony in a clean makefile.

Reply:
well that's a phylosopical questin if you should use makefiles only for build related stuff to generate artifacts or if you do "scripting" in it too for common tasks the project needs  | we do the later for a long time and should not break with it

PS: I have the `make vendor` in a local git hook for checkout, so vendor dir is always uptodate